.rubocop-https---shopify-github-io-ruby-style-guide-rubocop-yml

```diff
@@ -872,9 +872,6 @@ Lint/EmptyEnsure:
 Lint/EmptyInterpolation:
   Enabled: true
 
-Lint/EndInMethod:
```

Comment:
Rubocop was updated to 0.81.0 in CI (the gemspec does not constrain versions). This cop has been renamed to the (already-enabled) `Style/EndBlock` cop.